templates/injector-deployment.yaml

```
@@ -41,6 +41,8 @@ spec:
               value: {{ .Values.injector.logLevel | default "info" }}
             - name: AGENT_INJECT_VAULT_ADDR
               value: {{ include "vault.scheme" . }}://{{ template "vault.fullname" . }}.{{ .Release.Namespace }}.svc:{{ .Values.server.service.port }}
+            - name: AGENT_INJECT_VAULT_AUTH_PATH
+              value: {{ .Values.server.authPath }}
```

Should this be an injector parameter, instead of server?

```suggestion
              value: {{ .Values.injector.authPath }}
```